src/stories/Elements/Paragraph.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+import { storiesOf } from "@storybook/react";
+
+import Paragraph from "../../Elements/Paragraph";
+
```

Comment:
In order to make it more obvious how to use it, we should probably decide if we'd like to use `size="small/medium/large"` or `size="leading/normal/sub"`. I think we should aim for the Storybook to be a documentation on how to use it, so instead of "Leading" it should probably be called "Large" since that's what the prop correspond to?

Reply:
i agree @lessp, please see latest commit where i changed naming standard.